tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -4542,11 +4564,15 @@ def tailLoopAllocG2LVgpr(self, kernel):
     if not kernel["DirectToVgprA"]:
       if ("ULSGRODoubleG2L" in kernel) and kernel["ULSGRODoubleG2L"] == 1:
         numG2LA = self.states.a.numVgprG2LAllocated*2
+      elif kernel["DirectToLdsA"] and kernel["NonDTLTailLoop"]:
```

Comment:
I think this if statement is redundant.
You should be able to use numVgprG2LTailloopAllocated for any cases.

Reply:
Good point, I've switched to use `numVgprG2LTailloopAllocated` for all A/B cases in `tailLoopAllocG2LVgpr`.